### ISSUE TITLE
Tempo de aprovação do PR

### ISSUE THREAD

**Issue by rafaels88** [mathvbarone/awesome-programming-languages#5]
Recentemente o PR https://github.com/mathvbarone/awesome-programming-languages/pull/3 foi aberto e aparentemente foi dado o merge pouquissimo tempo depois. Na minha opinião, como esse repositório se trata do resultado de algumas discussões e não de um código certo/errado que passa num CI, os PRs deveriam ficar abertos um pouco mais de tempo para dar oportunidade a uma possível discussão em cima dele. Ao contrário, teremos sempre informações quase não debatidas já no `master`.

A proposta é simples: podemos ter um tempo, talvez pelo menos um dia, em cada PR aberto antes de ser dado o merge?

**Comment by mathvbarone** [comment]
@rafaels88  acho extremamente válido o seu ponto, e confesso que me precipitei em autorizar o PR. Vamos fazer assim a partir de agora todo PR vai ficar durante 24 horas aberto para possíveis discussões, o que acha?

**Comment by rafaels88** [comment]
Eu acho incrível a ideia. E tomara que mais gente comente aqui pra opinar tbm!

E sobre a precipitação, o repositório acabou de ser criado. Normal esse período de entendimento de como será a mecânica, né?

**Comment by mathvbarone** [comment]
Exatamente, até pq esse é o primeiro repo que fico como adm, então ainda to pegando as manhas nesse negócio de administrar os PRs ahahahaha.

**Comment by wilsonjuniordev** [comment]
Concordo plenamente, até por que temos a tendência de favorecer determinada linguagem que conhecemos e penso que o objetivo é tentar ser o mais imparcial possível. Mas com certeza vai ter várias alterações da mesma linguagem.
